test/unit/wallet.js

```diff
@@ -13,6 +13,7 @@ const common = require('../../src/common');
 const bitcoin = require('../../src/bitcoin');
 const should = require('should');
 const nock = require('nock');
+const sinon = require('sinon');
```

Comment:
This introduces up to 8 new dependencies. Are you sure you need `sinon`? Could you perhaps build this test with nock?

Reply:
sinon is already a dev-dependency in bitgojs and used extensively in other bitgojs tests

Reply:
I see, thanks for pointing that out.